### ISSUE TITLE
Action to deploy site is failing

### ISSUE THREAD

**Issue by crwilcox** [wmrra/wmrra-com#207]
Dion reached out to me and asked if I'd have a look. The following error is occuring.

```
Error: error building site: "/home/runner/work/wmrra-com/wmrra-com/content/sponsor/become-a-sponsor.md:8:1":
failed to render shortcode "incpartial": failed to process shortcode: "/home/runner/work/wmrra-com/wmrra-com/layouts/shortcodes/incpartial.html:1:3":
execute of template failed: template: shortcodes/incpartial.html:1:3:
executing "shortcodes/incpartial.html"
at <partial (.Get 0) .>: error calling partial: "/home/runner/work/wmrra-com/wmrra-com/layouts/partials/become-a-sponsor.html:10:47":
execute of template failed: template: partials/become-a-sponsor.html:10:47:
executing "partials/become-a-sponsor.html"
at <append (slice $row)>: error calling append: cannot append slice of []map[string]interface {} to slice of map[string]interface {}
```

This worked last time, Dion was updating the schedule, not sponsors, so it seemed unrelated.

I thought about the issue, Before we build we have to set the job up and install Hugo. I tried with my local version of Hugo (old as I haven't worked on the site in a while) and it worked. It doesn't work with the latest version of hugo (I tried 115).

Someone will need to investigate the sponsor partial and make it compatible with newer versions of Hugo.

Alternatively, someone could alter the infrastructure scripts and pin to an old version of Hugo. This would be a stop gap and shouldn't be considered a long term solution as it would result in a few issues, notable maintainability and security risks.

**Comment by NodiBlue** [comment]
I spent some time reviewing this with a couple different approaches. I've identified the build that broke the code (or updated it for several reasons). The last working build was v0.113.0. The next build, V0.114.0, exits the build with the same error codes as previously indicated.

Build v0.114.0 was released on 6/19/2023 (it's a big one). I did an initial comparison between v0.113.0 and v0.114.0, but the changes were many.

I'd prefer to update the code to align with the "latest" build as well, but unsure I can indicate what changed. Rolling back the build on the server isn't something I'm familiar with, but I clearly have too much time on my hands (I'll investigate/learn).

**Comment by NodiBlue** [comment]
For expedience I changed the "firebase_prod_deploy.yml" file to lock in the last working build. While this isn't optimal I didn't have the time to work through the code changes needed. The next round is also upon us and people will be looking for the schedule (although on the OMRRA website).

If/when I get more time I'll investigate the code changes required to support future Hugo releases.

This is my change of record (indicated in the change history):
```
From: 
hugo-version: 'latest'

To: 
hugo-version: '0.113.0'
```

Thank you @crwilcox

**Comment by praneetloke** [comment]
I think the issue is https://github.com/gohugoio/hugo/issues/11131. It's best described in a [comment](https://github.com/gohugoio/hugo/issues/11131#issuecomment-1598079671). It looks like it's tagged to be fixed in `0.116.0`. So I think pinning the version is the best solution for right now.
